airsonic-main/src/main/java/org/airsonic/player/controller/CoverArtController.java

```diff
@@ -150,7 +150,12 @@ private CoverArtRequest createCoverArtRequest(HttpServletRequest request) {
         if (id.startsWith(PODCAST_COVERART_PREFIX)) {
             return createPodcastCoverArtRequest(Integer.valueOf(id.replace(PODCAST_COVERART_PREFIX, "")), request);
         }
-        return createMediaFileCoverArtRequest(Integer.valueOf(id), request);
+        try {
+            return createMediaFileCoverArtRequest(Integer.valueOf(id), request);
+        } catch (java.lang.NumberFormatException x) {
+            LOG.warn("Invalid cover id: " + id, x);
```

Comment:
I would not want a full stack trace for these caught `NumberFormatException`s.  Maybe use `x.toString()` instead?